views/login.ejs

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
this is the same process as before... except I didn't end up changing the text inside of the mobile hidden navbar. This should be an easy fix if need be